pandas/parser.pyx

```diff
@@ -1750,6 +1772,78 @@ cdef inline int _try_double_nogil(parser_t *parser, int col,
 
     return 0
 
+cdef _try_uint64(parser_t *parser, int col, int line_start, int line_end,
+                 bint na_filter, kh_str_t *na_hashset):
```

Comment:
It's _possible_ I imagine but not as nice since the implementations are definitely customized to the `dtype` (e.g. note how I did not pass in any `NA` value for `uint64`, but the `int64` implementation does)